mycore-webtools/src/main/vue/processing/src/components/CollectionComponent.vue

```diff
@@ -20,122 +20,99 @@
 import ProcessableComponent from "./ProcessableComponent.vue";
 import CollectionModal from "./CollectionModal.vue";
-import {ref} from "vue";
+import {computed, ref, useTemplateRef} from "vue";
 
-defineProps<{ collection: Collection }>();
+const props = defineProps<{ collection: Collection }>();
 
 const selectedTab = ref<'processing' | 'waiting' | 'finished'>('processing');
 
-function toggleModal() {
-  isModalVisible.value = !isModalVisible.value;
+const collectionModal = useTemplateRef("collectionModal");
+
+function showModal() {
+  if (collectionModal.value) {
+    collectionModal.value.show();
+  }
 }
-const isModalVisible = ref(false);
+
+const currentTabCollection = computed(() => {
+  switch (selectedTab.value) {
+    case 'processing':
+      return props.collection.processingProcessables;
+    case 'waiting':
+      return props.collection.createdProcessables;
+    case 'finished':
+      return props.collection.finishedProcessables;
+  }
+});
 
 </script>
 
 <template>
-  <h2>
-    {{ collection.name }}
-    <a class="modal-button" href="#" @click.prevent="toggleModal">(Properties)</a>
-  </h2>
-  <!-- TAB -->
-  <div style="white-space: nowrap; overflow-x: auto;">
-    <button
-        @click="selectedTab = 'processing'"
-        :class="{ active: selectedTab === 'processing' }">
-      Processing <span class="badge">{{ collection.processingProcessables.length }}</span>
-    </button>
-    <button
-        @click="selectedTab = 'waiting'"
-        :class="{ active: selectedTab === 'waiting' }">
-      Waiting <span class="badge">{{ collection.createdProcessables.length }}</span>
-    </button>
-    <button
-        @click="selectedTab = 'finished'"
-        :class="{ active: selectedTab === 'finished' }">
-      Finished <span class="badge">{{ collection.finishedProcessables.length }}</span>
-    </button>
-  </div>
-  <!-- HEADER -->
-  <div class="table-header">
-    <div class="col col-name">Name</div>
-    <div class="col col-user">User</div>
-    <div class="col col-create">Create Date</div>
-    <div class="col col-progress">Progress</div>
-  </div>
-  <!-- CONTENT -->
-  <div class="table-content">
-    <div v-if="selectedTab === 'processing'">
-      <ProcessableComponent v-for="processable in collection.processingProcessables"
-                            :model="processable"
-                            :key="processable.id"/>
-      <div v-if="collection.processingProcessables.length === 0">
-        no active process
-      </div>
-    </div>
-    <div v-if="selectedTab === 'waiting'">
-      <ProcessableComponent v-for="processable in collection.createdProcessables"
-                            :model="processable"
-                            :key="processable.id"/>
-      <div v-if="collection.createdProcessables.length === 0">
-        no waiting process
-      </div>
+  <div class="card mb-4">
+    <div class="card-header">
+
+      <h2 class="d-inline h5">{{ collection.name }}</h2>
```

Comment:
h2 with a look of h5 seems to be not the best solution

Reply:
Why?

Reply:
I agree with @kkrebs - why not `<h5>` directly

Reply:
Accessibility requires the use of h1 to h{n} without a gap. So this header has to be in a chapter of h{n-1}. So I see it like @sebhofmann here. See https://sa11y.netlify.app/ for a bookmarklet to check for accessibility and site structure.

Reply:
https://www.a11yproject.com/posts/how-to-accessible-heading-structure/